components/Navigation/index.js

```diff
@@ -0,0 +1,40 @@
+import { styled } from "styled-components";
+import Link from "next/link";
+import { device } from "@/utils/globalValues";
+
+export default function Navigation() {
+  return (
+    <StyledNavigation>
+      <li className="inherit-background-color">
+        <StyledNavigationLink href="/words">Words</StyledNavigationLink>
+      </li>
+      <li className="inherit-background-color">
+        <StyledNavigationLink href="/add">Add</StyledNavigationLink>
+      </li>
+    </StyledNavigation>
+  );
+}
+
+const StyledNavigation = styled.footer`
```

Comment:
```suggestion
const StyledNavigation = styled.nav`
```
A [navigation](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav) that's at the bottom of the screen is still a navigation.